packages/arb-token-bridge-ui/tests/e2e/specs/withdraw.cy.ts

```diff
@@ -0,0 +1,173 @@
+      cy.login()
+    })
+
+    beforeEach(() => {
+      // restore local storage from first test
+      cy.restoreLocalStorage()
+    })
+
+    afterEach(() => {
+      // cypress clears local storage between tests
+      // so in order to preserve local storage on the page between tests
+      // we need to use the cypress-localstorage-commands plugin
+      // or else we have to visit the page every test which will be much slower
+      // https://docs.cypress.io/api/commands/clearlocalstorage
+      cy.saveLocalStorage()
+    })
+
+    it('should show L1 and L2 ETH balances correctly', () => {
+      cy.findByText(`Balance: ${l1ETHbal} ETH`).should('be.visible')
+      cy.findByText(`Balance: ${l2ETHbal} ETH`).should('be.visible')
+    })
+
+    it('should switch to L2 network correctly', () => {
+      cy.findByRole('button', { name: /Switch Networks/i })
+        .should('be.visible')
+        .click({ scrollBehavior: false })
+        .then(() => {
+          cy.findByRole('button', { name: /From: Arbitrum Goerli/i }).should(
+            'be.visible'
+          )
+          cy.findByRole('button', { name: /To: Goerli/i }).should('be.visible')
+
+          cy.findByRole('button', {
+            name: /Move funds to Goerli/i
+          })
+            .should('be.visible')
+            .should('be.disabled')
+        })
+    })
+
+    it('should show empty bridging summary', () => {
+      cy.findByText('Bridging summary will appear here.').should('be.visible')
+    })
+
+    context("bridge amount is lower than user's L2 ETH balance value", () => {
+      const zeroToLessThanOneETH = /0(\.\d+)*( ETH)/
+      it('should show summary', () => {
+        cy.findByPlaceholderText('Enter amount')
+          .type('0.0001', { scrollBehavior: false })
+          .then(() => {
+            cy.findByText('You’re moving')
+              .siblings()
+              .last()
+              .contains('0.0001 ETH')
+              .should('be.visible')
+            cy.findByText('You’ll pay in gas fees')
+              .siblings()
+              .last()
+              .contains(zeroToLessThanOneETH)
+              .should('be.visible')
+            cy.findByText('L1 gas')
+              .parent()
+              .siblings()
+              .last()
+              .contains(zeroToLessThanOneETH)
+              .should('be.visible')
+            cy.findByText('L2 gas')
+              .parent()
+              .siblings()
+              .last()
+              .contains(zeroToLessThanOneETH)
+              .should('be.visible')
+            cy.findByText('Total amount')
+              .siblings()
+              .last()
+              .contains(/(\d*)(\.\d+)*( ETH)/)
+              .should('be.visible')
+          })
+      })
+
+      it('withdraw button should be clickable', () => {
+        cy.findByRole('button', {
+          name: /Move funds to Goerli/i
+        })
+          .should('be.visible')
+          .should('not.be.disabled')
```

Comment:
i don't have a strong opinion on this as `not.be.disabled` works
but what do you think of using `be.enabled` instead?

https://docs.cypress.io/guides/references/assertions#Disabled-property